docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -891,7 +891,10 @@ export interface SdkHttpOperation {
   kind: "http";
   // raw HTTP operation output from typespec/http
   __raw: HttpOperation;
+  // route path for the target operation. TypeSpec suggest to use `uriTemplate` instead as `path` will not work for complex cases like not-escaping reserved chars.
```

Comment:
Should we deprecate path? Or wait until typespec deprecated path as well

Reply:
after typespec, i believe.